app/assets/stylesheets/components/_countdown.scss

```
@@ -0,0 +1,26 @@
+ul#booking-show-countdown {
+  list-style: none;
+  margin: 50px 0;
+  padding: 0;
+  display: block;
+  text-align: center;
+}
+
+ul#booking-show-countdown li { display: inline-block; }
+
+ul#booking-show-countdown li span {
+  font-size: 40px;
+  font-weight: 300;
+  line-height: 80px;
+}
+
+ul#booking-show-countdown li.seperator {
```

typo on `separator`, I would recommend changing in here and in the show.html file.
```suggestion
ul#booking-show-countdown li.separator {
```